pybamm/solvers/scikits_ode_solver.py

```diff
@@ -14,6 +14,11 @@
     if scikits_odes_spec is not None:
         scikits_odes = importlib.util.module_from_spec(scikits_odes_spec)
         scikits_odes_spec.loader.exec_module(scikits_odes)
+        # NOTE: Gives error module 'scikits.odes' has no attribute 'sundials'
+        # if you try to make the Jacobian class using
+        # scikits_odes.sundials.cvode.CV_JacRhsFunction
+        # but is OK if you import cvode here?
+        from scikits.odes.sundials import cvode
```

Comment:
again @martinjrobins

Reply:
I think this is fine (or at least out of our hands)